### DISCUSSION TITLE
ALLOW REVOLT.SOCIAL TO BE AN OFFICIAL PARTNER AGAIN.

### DISCUSSION THREAD

**Discussion by Adithr123** [stoatchat/discussions#281] [Feature Suggestions]
revolt.social was amazing, but it got shut down. Please bring it back.

**Comment by insertish** [comment]
This is a lot more complicated than just running as a partner, the whole site was pretty much taken down in its entirety without prior notice.

Edit: I wouldn't normally be here at 5am but I happen to be awake for unrelated reason

(Also we have stuff being built already as a replacement)

**Comment by Adithr123** [comment]
Ok. Thank you! 😊

**Comment by nerrixde** [comment]
@Adithr123 @insertish revolt.social will not come back. Their new discovery looks amazing https://rvlt.gg/discover/servers https://rvlt.gg/discover/bots
